index.js

```diff
@@ -256,7 +256,7 @@ module.exports = function(options) {
     while (m.parent) {
       // The test file is the root as far as we are concerned,
       // not mocha itself
-      if (m.parent.filename.match(/\/node_modules\/mocha\//)) {
+      if (m.parent.filename.match(new RegExp("node_modules" + path.sep + "mocha"))) {
```

Comment:
I am not sure this is relevant to the same issue. Also it is not specific enough. It would match "foonode_modules" or "mochaish" because it is lacking the leading and trailing slashes.

Reply:
I understand, I believe I should make it `new RegExp(path.sep + "node_modules" + path.sep + "mocha" path.sep)` to maintain the path specifically for inside `node_modules` folder with `mocha` path. Is that okay? @boutell